openlibrary/templates/covers/author_photo.html

```diff
@@ -15,12 +17,14 @@
                 <h2>$truncate(author.name or "", 60)</h2>
                 <a class="dialog--close">&times;<span class="shift">$_("Close")</span></a>
             </div>
-            <a class="dialog--close" title="Click to close"><img src="$author_photo_url" class="cover" alt="Photo of $author.name"/></a>
+            <a class="dialog--close" title="Click to close">
+                <img src="$author_photo_url" itemprop="image" class="cover" alt="Photo of $author.name"/>
+            </a>
         </div>
     </div>
 $else:
     <div class="author coverMagic">
         <div class="SRPCover bookCover">
-            <img src="/images/icons/avatar_author-lg.png" class="cover" alt="We need a photo of $author.name"/>
+            <img src="/images/icons/avatar_author-lg.png" itemprop="image" class="cover" alt="We need a photo of $author.name"/>
```

Comment:
In this case, since the image is just a placeholder image, we likely _don't_ want it to be itemprop=image
```suggestion
            <img src="/images/icons/avatar_author-lg.png" class="cover" alt="We need a photo of $author.name"/>
```

Reply:
Will remove it :+1: